### DISCUSSION TITLE
Customizing formatting

### DISCUSSION THREAD

**Discussion by nihiluis** [godotengine/godot-vscode-plugin#637] [Q&A]
Can I do a config file for this plugin? The web community has .prettierrc for example.

One thing (among others) I dislike is the insertion of a space between preload and the param here:
`const Reputation7Texture = preload ("res://scenes/ui/icons/agent/reputation7.png")`

**Comment by DaelonSuzuka** [comment]
Making a configurable formatter is easily 10x the amount of work, so I'm probably not going to do that.

> One thing (among others) I dislike is the insertion of a space between preload and the param here:
`const Reputation7Texture = preload ("res://scenes/ui/icons/agent/reputation7.png")`

Yeah, that's definitely a bug.

**Comment by DaelonSuzuka** [comment]
The extra space inserted into `preload()` has been fixed in #605.
